CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Change Log
 
+## v0.11.0
+
+([full changelog](https://github.com/executablebooks/jupyter-book/compare/v0.10.2...d43216c57a2710eaeb2363f21d2b3aadf914feca)) | [GitHub contributors page for this release](https://github.com/executablebooks/jupyter-book/graphs/contributors?from=2021-04-04&to=2021-05-11&type=c)
+
+This is primarily a documentation and under-the-hood update, but it comes with an important change to the way that **Table of Contents** are structured. See below for more information.
```

Comment:
Ah you can get it more hype than this 😄. Maybe mention about the globbing feature, and that you can now use other file formats (basically mentioning some of the issues that the toc PR closed)
Also might want to specifically mention that the `jupyter-book toc` CLI command has changed to a group (and the original command is now equivalent to `jupyter-book toc from-site`)

Reply:
I agree - I'm just working on very limited bandwidth right now. I will try and get to this and flesh out the changelog a bit but it'll depend on baby / other work duties.

Reply:
Yeh no worries 👍

Reply:
@choldgraf I will try and take a look at this today.